resources/views/blueprints/index.blade.php

```diff
@@ -179,6 +179,24 @@
         </table>
     </div>
 
+    @foreach ($additional as $namespace)
+    <h3 class="little-heading pl-0 mb-2">{{ $namespace['title'] }}</h3>
+    <div class="card p-0 mb-4">
+        <table class="data-table">
+        @foreach ($namespace['blueprints'] as $blueprint)
+            <tr>
+                <td>
+                    <div class="flex items-center">
+                        <div class="w-4 h-4 mr-4">@cp_svg('icons/light/users')</div>
```

Comment:
A generic one might be simpler as otherwise you'd either need to add an icon() method to the blueprint class or add an icon when adding a namespace so all blueprints in the namespace have the same icon. 

Do we just run with https://github.com/statamic/cms/blob/4.x/resources/svg/icons/light/blueprint.svg for now?

Reply:
Yeah, I think that might be the best one to use.

Reply:
Done